tests/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -135,14 +135,17 @@ public interface IFoo
                 .x(() => fake = A.Fake<FakedClass>());
 
             "When I get the property value"
-                .x(() => result = fake.FakeableProperty);
+                .x(() => result = fake.FakeableProperty!);

```

Comment:
Maybe change `result` to `IFoo`?

Reply:
Of course, that means you'll have to do `result!.IsADummy` later...

Reply:
I think it would be better to use `!` on `result!.IsADummy`, because as this point we will already have checked that it's not null.

Reply:
Done!

Reply:
👍